## <center>Hypothesis Testing1</center>

#### Problem Statement:
<code style="background:lightblue;color:black">A F&B manager wants to determine whether there is any significant difference in the diameter of the cutlet between two units. A randomly selected sample of cutlets was collected from both units and measured? Analyze the data and draw inferences at 5% significance level. Please state the assumptions and tests that you carried out to check validity of the assumptions.</code>

In [1]:
#Importing the necessary libraries for the model
import pandas as pd
from scipy import stats

In [2]:
#Importing the csv file and having a glance at the data
cutlet_data=pd.read_csv("Cutlets.csv")
cutlet_data.head()

Unit A  Unit B
0  6.8090  6.7703
1  6.4376  7.5093
2  6.9157  6.7300
3  7.3012  6.7878
4  7.4488  7.1522

In [3]:
#Forming a list
Unit_A=cutlet_data['Unit A']
Unit_B=cutlet_data['Unit B']

In [4]:
#Finding the pvalue
pval = stats.ttest_ind(Unit_A,Unit_B)[1]
pval

0.4722394724599501

In [5]:
#Printing the output to know wether to accept the null hypothesis or not
if pval < 0.05:
    print('we reject the null hypotheis')
else:
    print('we accept the null hypothesis')

we accept the null hypothesis


#### Insights Drawn:
> Here H0: There is no significant difference in the diamaters of unit A and unit B (diameteres of the cutlets are same)<br>  H1:There is a significant difference in the diamaters of unit A and unit B (diameteres of the cutlets are not same)<br> Hence From the pvalue above it is evident that we are accepting the null hypothesis since Pvalue=0.472>0.05,so there is no significant difference in the diameters of the cutlet.